README.md

```diff
@@ -78,6 +78,7 @@ File URIs support VSCode variables like: `${userHome}`. It just replaces support
 
 - `${cwd}`
 - `${userHome}`
+- `${workspaceFolder}`
```

Comment:
Add a note: currently we inject the contents of the CSS or JS file into the patched HTML, so this variable will meaning the first opening (or current? not tested) window's workspace path when applying CSS/JS